ccd-definition/CaseEvent/CareSupervision/StateChange.json

```diff
@@ -14,7 +14,6 @@
     "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/case-submission/about-to-submit",
     "RetriesTimeoutURLAboutToSubmitEvent": "1,2,3,4,5",
     "CallBackURLSubmittedEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/case-submission/submitted",
-    "RetriesTimeoutURLSubmittedEvent": "1,2,3,4,5",
```

Comment:
I dont think that this is what you want to achieve. I guess it will cause default retries to be used. Have you tested this behaviour?

Reply:
i think you should use:
"RetriesTimeoutURLSubmittedEvent": "0",

Reply:
Thanks

Reply:
what about C2 payment?